tests/Functional/src/com/microsoft/aad/adal/test/DefaultTokenCacheStoreTests.java

```diff
@@ -74,7 +74,7 @@ protected void tearDown() throws Exception {
     }
 
     public void testSharedCache() throws GeneralSecurityException, IOException {
```

Comment:
Can you rename it to something more appropriate, for me it looks like testGetItem